test/e2e/down.spec.js

```diff
@@ -10,7 +10,7 @@ describe('teardown', () => {
     expect(await utils.getServiceVersion('one-two.yml', 'two')).to.equal('1.0.0');
     expect(await utils.getServiceVersion('three.yml', 'three')).to.equal('1.0.0');
 
-    await utils.serviceComposeCommand('down --remove-orphans -t 1');
+    await utils.serviceComposeCommand('rm -fs');
```

Comment:
`down` will also try to remove the network and it will fail, since it has active endpoints.